spec/controllers/content_types_controller_spec.rb

```diff
@@ -18,9 +18,7 @@
       'version' => 1,
       'type' => content_type,
       'externalIdentifier' => pid,
-      'access' => {
-        'access' => 'world'
-      },
+      'access' => {},
```

Comment:
You'll see this change in many places. It is just using some valid value (in this case, the default dark), rather than an invalid value (world access without a download attribute). The actual valid value doesn't matter.